docs/gameobject.md

```diff
@@ -18,63 +18,65 @@ This table holds the individual object data on each spawned game object in the w
+| [position_y][8]     | FLOAT        | SIGNED     |     | NO   | 0       |                |                          |
+| [position_z][8]     | FLOAT        | SIGNED     |     | NO   | 0       |                |                          |
+| [orientation][9]    | FLOAT        | SIGNED     |     | NO   | 0       |                |                          |
+| [rotation0][10]     | FLOAT        | SIGNED     |     | NO   | 0       |                |                          |
+| [rotation1][10]     | FLOAT        | SIGNED     |     | NO   | 0       |                |                          |
+| [rotation2][10]     | FLOAT        | SIGNED     |     | NO   | 0       |                |                          |
+| [rotation3][10]     | FLOAT        | SIGNED     |     | NO   | 0       |                |                          |
+| [spawntimesecs][11] | INT          | SIGNED     |     | NO   | 0       |                |                          |
+| [animprogress][12]  | TINYINT      | UNSIGNED   |     | NO   | 0       |                |                          |
+| [state][13]         | TINYINT      | UNSIGNED   |     | NO   | 1       |                |                          |
 
 [1]: #guid
 [2]: #id
 [3]: #map
-[4]: #zoneId
-[5]: #areaId
+[4]: #ZoneId
+[5]: #AreaId
 [6]: #spawnmask
 [7]: #phasemask
-[8]: #position_x
-[9]: #position_y
-[10]: #position_z
-[11]: #orientation
-[12]: #rotation0
-[13]: #rotation1
-[14]: #rotation2
-[15]: #rotation3
-[16]: #spawntimesecs
-[17]: #animprogress
-[18]: #state
-
-**Description of the fields**
-
-### guid
+[8]: #position
+[9]: #orientation
+[10]: #Rotation
+[11]: #spawntimesecs
+[12]: #animprogress
+[13]: #state
+
+## Column's description
+
+### Guid
 
 The global unique identifier for the game object. This field must be unique among all game objects.
 
-### id
+### Id
 
 The template ID of the gameobject. See [gameobject\_template.entry](http://www.azerothcore.org/wiki/gameobject_template#entry)
 
-### map
+### Map
 
-The map ID where this object is spawned. See Maps.dbc
+The map ID where this object is spawned.The most used values are:
+Map ID | Map name
+--- | ---
+0 | Eastern Kingdoms
+1 | Kalimdor
+530 | Northrend
 
-### zoneId
+For more information, use the [DBC Reader](https://github.com/wowgaming/node-dbc-reader) on Maps.dbc for the used data.
+
+### ZoneId
 
 The ID of the zone that this object is spawned in. (e.g. The Barrens)
 
 This column is filled in by the worldserver on startup if the `Calculate.Gameoject.Zone.Area.Data` setting is enabled. It originates from AreaTable.dbc.
 
-### areaId
+### AreaId
 
 The ID of the area that this object is spawned in. You can think of an area as a "subzone" of a zone, e.g. Lushwater Oasis inside The Barrens. 
 
 This column is filled in by the worldserver on startup if the `Calculate.Gameoject.Zone.Area.Data` setting is enabled. It originates from AreaTable.dbc.
 
-### spawnMask
+### SpawnMask
```

Comment:
Wrong. In the db they are named spawnMask. several places you have changed this incorrectly.